### PR TITLE
uploaded arideda function

### PR DESCRIPTION
**Pull request by nphaterp**

### CODE REVIEW

File: R/aridanalysis.R [hunk 1]
```diff
@@ -1,3 +1,11 @@
+library(testthat)
+library(tidyverse)
+library(palmerpenguins)
+library(ggplot2)
+library(GGally)
+library(grid)
+source("http://peterhaschke.com/Code/multiplot.R")
```
Comment (CraigEnsemble) [inline R/aridanalysis.R:7]: I think we'll need to have this in our function and not our source

File: R/eda_tests.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
+test_that('Categorical plot should be denisty plots based on selected features ', {
```
Comment (CraigEnsemble) [inline R/eda_tests.R:1]: Small typo

File: R/aridanalysis.R (hunk 1 above)
Comment (cmmclaug) [inline R/aridanalysis.R:7]: I don't think we can source this into the package, and we probably shouldn't use a URL for this. Bring code into function.

File: R/eda_tests.R (hunk 2 above)
Comment (cmmclaug) [inline R/eda_tests.R]: I think this needs to be in tests/testthat.

Comment (cmmclaug) [inline R/eda_tests.R]: Small misspelling of density

**Review by cmmclaug — CHANGES_REQUESTED** [review]

**Pull request merged**